operations/iuf/workflows/preparation.md

```diff
@@ -0,0 +1,48 @@
+# Prepare for the install or upgrade
+
+This section defines environment variables and directory content that is used throughout the workflow.
+
+**`NOTE`** The following step uses the `iuf activity` command to demonstrate how to record operations within an IUF activity. While `iuf` automatically records all `iuf run` operations within an IUF activity, any other
+administrative operation can also be recorded within an IUF activity by using `iuf activity` in this manner. The remainder of the workflow will not use `iuf activity`, deferring to the administrator to use it as they desire.
```

Comment:
...administrative operation**s** (make plural)

Reply:
Fixed in next commit.